src/ui/public/routes/__tests__/_work_queue.js

```diff
@@ -3,7 +3,7 @@ import expect from 'expect.js';
 import ngMock from 'ng_mock';
 import { WorkQueue } from 'ui/routes/work_queue';
 import sinon from 'auto-release-sinon';
-import 'ui/promises';
+import 'ui/promises/index';
```

Comment:
This should not be necessary since: https://github.com/elastic/kibana/commit/9bdbc260c6f286e5d1af8caae808b63d8be7e337

If for some reason it is please file an issue at https://github.com/elastic/webpack-directory-name-as-main, but if we remove `promiseMapSettled()` then this doesn't really mean much.